lib/react_on_rails/packs_generator.rb

```diff
@@ -23,13 +23,11 @@ def self.raise_nested_entries_disabled
     def verify_setup_and_generate_packs
       return unless components_subdirectory.present?
 
-      raise_webpacker_not_installed unless ReactOnRails::WebpackerUtils.using_webpacker?
-      raise_shakapacker_version_incompatible unless shackapacker_version_requirement_met?
-      raise_nested_entries_disabled unless ReactOnRails::WebpackerUtils.nested_entries?
+      verify_configuration
 
       is_generated_directory_present = Dir.exist?(generated_packs_directory_path)
 
-      return if is_generated_directory_present && webpack_assets_status_checker.stale_generated_component_packs.empty?
+      return if is_generated_directory_present && stale_or_missing_packs.empty?
```

Comment:
is the stale_or_missing_packs list used to optimize what's generated?

Reply:
If not, then why keep going after one is detected?

Reply:
This optimization should be a separate PR as it requires additional testing and might introduce new bugs if not done correctly. Considering the time sensitivity of this work, I would suggest creating a new Issue for optimization, which we can work on later.

Reply:
then exit early on the first found stale or missing entry

Reply:
We will use this list later to generate packs just for them. I can exit it early by making the `stale_or_missing_packs` return boolean. But since we are going to use it very soon, I suggest keeping it this way for now.